### DISCUSSION TITLE
Welcome to gaszaehler-home-assistant-esphome discussions!

### DISCUSSION THREAD

**Discussion by svenwal** [svenwal/gaszaehler-home-assistant-esphome#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by doczkal** [comment]
I could not find what to by for the reed sensor. Can you provide more information?

**Reply by svenwal** [reply]
For example https://amzn.eu/d/6tZffCt

**Reply by doczkal** [reply]
Thanks.
